central/graphql/resolvers/clusters.go

```diff
@@ -80,6 +87,8 @@ func init() {
 			"subject(name: String!): Subject",
 			"subjectCount(query: String): Int!",
 			"unusedVarSink(query: String): Int",
+
+			"plottedVulns(query: String): PlottedVulnerabilities!", // TODO
```

Comment:
intentionally out-of-order?

Reply:
Yeah I pulled plottedvulns out of the ordering since it will not be included in new resolvers and will be deprecated in existing resolvers once the plottedvulns work is completed